src/org/openlcb/hub/Hub.java

```diff
@@ -37,20 +37,41 @@ public class Hub {
     private final static Logger logger = Logger.getLogger(Hub.class.getName());
     public final static int DEFAULT_PORT = 12021;
     final static int CAPACITY = 20;  // not too long, to reduce delay
-    private final boolean lineEndings;
+    private final boolean sendLineEndings;
+    private final boolean requireIncomingLineEndings;
     private boolean disposed = false;
 
+    /**
+     * Constructs a new instance using default values.
+     * Port - 12021
+     * Send line endings - true
+     * Require incoming line endings - true
```

Comment:
I liked it better in the previous iteration of the PR when we had require incoming line endings == false as default.
This provides the most compatibility.

I understand that this is a change in behavior from the previous code, but I believe wider compatibility is better, so I will approve this change in behavior.

Reply:
No problem, will change the default back to the new version.